Applications/SlicerApp/Testing/Python/CMakeLists.txt

```diff
@@ -234,6 +234,12 @@ slicer_add_python_unittest(
   TESTNAME_PREFIX nomainwindow_
   )
 
+slicer_add_python_unittest(
+  SCRIPT ${Slicer_SOURCE_DIR}/Base/Python/slicer/tests/test_slicer_util_getNodes.py
+  SLICER_ARGS --no-main-window --disable-cli-modules --disable-scripted-loadable-modules DATA{${INPUT}/MR-head.nrrd}
```

Comment:
Instead of loading a file, could you look into creating a node in the setup method.

For example, you could do:

```
slicer.mrmlScene.AddNode(slicer.vtkMRMLScalarVolumeNode())
```